scripts/triggers-end-to-end-tests/functions/.eslintrc.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Let's not add new linting for the end-to-end test source.  We can either ignore it or extend the root `lint` task to look at these files.